msteams-platform/concepts/deploy-and-publish/overview.md

```diff
@@ -1,56 +1,72 @@
 ---
 title: Distribute your app
-description: Describes the three options for distributing your app`
+description: Describes the three options for distributing your app
 ms.topic: conceptual
 keywords: teams publish store office distribute AppSource sideload upload app
 ---
-# Distribute your Microsoft Teams app
 
-Once you've created your app there are three options for distributing it:
+# Distribute your app
+This document guides you on the methods you can use to distribute your app to different organizations, such as Enterprise and US Government. 
 
-1. [Upload your app directly](#upload-your-app-directly).
-2. [Publish your app to your organization's app catalog](#publish-to-your-organizations-app-catalog).
-3. [Publish your app through AppSource](#publish-to-appsource).
+For Enterprise organizations you can directly upload your app or publish it your organization's app catalog, or publish it through the AppSource. For Government organizations, they need to be members of Government Cloud Community (GCC). Depending on the fulfillment of the GCC compliance criteria, the GCC tenant administrator decides if an app must be uploaded and published.
```

Comment:
I think this doc needs to be structured based on the available options to match publish workflows specified elsewhere. We can provide specific guidance (enterprise vs. GCC) within each, if applicable. What do you think of this?

- Intro
- Preview your app in Teams (upload directly)
- Publish to your organization
  - Configure a Government Community Cloud (GCC) distribution
- Publish to the Teams store

Reply:
@heath-hamilton - have updated it per your suggestion, but would request you to have a relook at the section to ensure we haven't missed out anything.